interfaces/Portalicious/src/app/pages/auth-callback/auth-callback.page.ts

```diff
@@ -0,0 +1,18 @@
+import { NgComponentOutlet } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+
+import { AuthService } from '~/services/auth.service';
+
+@Component({
+  selector: 'app-auth-callback',
+  standalone: true,
+  imports: [NgComponentOutlet],
+  templateUrl: './auth-callback.page.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AuthCallbackPageComponent {
+  private authService = inject(AuthService);
+
+  CallbackComponent = this.authService.CallbackComponent;
```

Comment:
Minor detail but I think the AuthCallbackPageComponent should be the one retrieving the returnUrl, and passing it on to the callback component